lms/static/scripts/frontend_apps/components/dashboard/test/DashboardActivityFilters-test.js

```diff
@@ -257,6 +257,35 @@ describe('DashboardActivityFilters', () => {
       );
       assert.equal(getSelectContent(wrapper, 'students-select'), '2 students');
     });
+
+    it('filters every dropdown by the values selected in the rest', () => {
```

Comment:
```suggestion
    it('filters each dropdown by the values selected in the other dropdowns', () => {
```

I rephrased this to be slightly clearer about what "the rest" refers to.